static/thing.js

```diff
@@ -0,0 +1,21 @@
+class Thing {
+    constructor(fill = 'rgba(0,255,0,.6)') {
+        this.keyframes = []; // List of boxes corresponding to keyframes
+        this.fill = fill
```

Comment:
You're right. Later we'll change the fill to be based off of its class. This can be placeholder for now